src/perl/dmtest/DMTest/Separated.pm

```diff
@@ -24,22 +25,32 @@ my $log = Log::Log4perl->get_logger(__PACKAGE__);
 sub suite {
   my ($package) = assertNumArgs(1, @_);
 
+  my @testNames = ();
+  my $machine = undef;
   my $options = $package->makeDummyTest();
-  my $testList;
-  if (defined($options->{dmtestName})
-      && ($options->{dmtestName} ne "")
-      && ($options->{dmtestName} !~ /[\[\]{}*?]/)) {
-    # If unitTestName names isn't a pattern but appears to name exactly one
-    # test, just use it. If the name is bad, we'll find out soon enough.
-    $testList = $options->{dmtestName};
-  } else {
-    # TODO: Get all tests from dmtest
-    $testList = "vdo";
-  }
```

Comment:
Note that by removing this default, DMTest/Separated will default to running all tests defined in dmtest.

You probably want to update dmtests.suites to pass " --dmTesName =vdo"  as part of the Full alias like the other test aliases do.

Reply:
fixed